crates/holochain_sqlite/src/schema.rs

```diff
@@ -130,9 +117,84 @@ impl Migration {
         Ok(())
     }
 
-    pub fn run(&self, _conn: &mut Connection) -> rusqlite::Result<()> {
-        unimplemented!("actual migrations not yet implemented")
+    pub fn run_forward(&self, conn: &mut Connection) -> rusqlite::Result<()> {
+        conn.execute_batch(&self.forward)?;
```

Comment:
Maybe this should be done in an explicit transaction, to ensure it can roll-back on a crash or something?